src/components/TableView/TableView.tsx

```diff
@@ -1,50 +1,49 @@
-/* @flow */
-import * as React from 'react';
+import React from 'react';
 import { View } from 'react-native';
 
 import TableViewHeader from './TableViewHeader';
 import TableViewFooter from './TableViewFooter';
 import { withTheme } from '../../';
-import type { Theme } from '../../types/Theme';
-import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
+import { Theme } from '../../types/Theme';
+import { ViewStyle } from 'react-native';
 
 type Props = {
   /**
    * Provided by the ThemeProvider
    */
-  theme: Theme,
+  theme: Theme;
   /**
    * Header of TableView
    */
-  header?: string,
+  header?: string;
   /**
    * Indicates whether render empty header or not
    */
-  withoutHeader?: boolean,
+  withoutHeader?: boolean;
   /**
    * Optional styles of header
    */
-  headerStyle?: ViewStyleProp,
+  headerStyle?: ViewStyle;
```

Comment:
Style props need to be like:
```suggestion
  headerStyle?: StyleProp<ViewStyle>;
```